gather-scheduler.js

```diff
@@ -112,6 +114,15 @@ class GatherScheduler {
       })
       .then(_ => artifacts);
   }
+
+  static saveAssets(tracingData, url) {
+    const date = new Date();
+    const hostname = url.match(/^.*?\/\/(.*?)(:?\/|$)/)[1];
```

Comment:
Heh. I had that previously but it was pretty gross for a simple string extraction